.github/workflows/linting.yml

```
@@ -12,6 +12,8 @@ jobs:
   linting:
     name: Linting
     runs-on: ubuntu-22.04
+    permissions:
+      contents: write
```

```suggestion
      contents: write # for stefanzweifel/git-auto-commit-action to create a PR
```

Updating to match the other places we've given write permission.